components/browser/errorpages/src/main/assets/errorPageScripts.js

```diff
@@ -43,11 +43,11 @@ var advancedVisible = false;
  * Used to show or hide the "advanced" button based on the validity of the SSL certificate
  */
 function updateShowSSL(queryMap) {
-    const showSSL = queryMap.showSSL
+    const showSSL = queryMap.showSSL;
```

Comment:
We should have some comment indicating the intended values of this variable.
```suggestion
    /** @type {'true' | 'false'} */
    const showSSL = queryMap.showSSL;
```